packages/components/jest/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { RenderResult } from '@testing-library/react-native'
```

Comment:
1. Seems like the folder/file should be swapped? `utils/jest.ts` as it's somewhat similar to `storybook.tsx` of utils for a tooling set
2. Add to `.npmignore`